recipe/menu-v2.json

```
@@ -0,0 +1,32 @@
+{
+    "$schema": "https://json-schema.org/draft-07/schema",
+    "$id": "https://schemas.conda.io/menuinst-1.schema.json",
+    "menu_name": "Anaconda ({{ DISTRIBUTION_NAME }})",
+    "menu_items": [
+        {
+            "name": {
+                "target_environment_is_base": "Jupyter Notebook",
+                "target_environment_is_not_base": "Jupyter Notebook ({{ ENV_NAME }})"
+            },
+            "description": "Jupyter notebook launcher",
+            "activate": false,
+            "terminal": false,
+            "icon": "{{ MENU_DIR }}/jupyter.{{ ICON_EXT }}",
+            "command": [
+                "{{ BASE_PYTHON }}",
+                "{{ BASE_PREFIX }}/cwp.py",
+                "{{ PREFIX }}",
+                "{{ PYTHON }}",
+                "{{ SCRIPTS_DIR }}/jupyter-notebook-script.py",
+                "%USERPROFILE%"
```

```suggestion
                "\"%USERPROFILE%\""
```
https://github.com/AnacondaRecipes/notebook-feedstock/blob/menuinst2/recipe/menu-v1.json#L6
Should escape quotes be added around userprofile like in v1?

---

That shouldn't be necessary. `menuinst` does the quoting for us.

https://github.com/conda/menuinst/blob/df2237fa6f3c78d8148928b1bcbef7896eabdf00/menuinst/platforms/win.py#L325-L328

https://github.com/conda/menuinst/blob/df2237fa6f3c78d8148928b1bcbef7896eabdf00/menuinst/utils.py#L138-L155